com.unity.multiplayer.mlapi/Editor/CodeGen/NetworkBehaviourILPP.cs

```diff
@@ -235,6 +238,11 @@ private bool ImportReferences(ModuleDefinition moduleDefinition)
                         m_NetworkManager_ntable_FieldRef = moduleDefinition.ImportReference(fieldInfo);
                         m_NetworkManager_ntable_Add_MethodRef = moduleDefinition.ImportReference(fieldInfo.FieldType.GetMethod("Add"));
                         break;
+                    //IF TOOLS
```

Comment:
In this case I would suggest only merging this when we have the conditional dependency figure out.

Reply:
We can update later. This will be in develop until then, and won't be on release path right away

Reply:
I'm having a hard time getting comfortable with this with the knowledge I have and can get from this code.  Evidently this means "we only want this code when we are tools".  I'm not sure what "tools" means.  Does it mean "when running in the editor"?

If we really need to get this in before we sort out what this means / how we'll block it, I could live with 

```suggestion
   #ifdef TOOLS_MODE
```

Where TOOLS_MODE is always false, and then for your development in the meantime you can force it on, vs. having it default on with a TODO to remember to block it later (opt-in now vs. future opt-out).  Maybe I'm over paranoid, but remember develop is used now by early-adopter devs.

Reply:
Why are we trying to strip out this code? It is a static lookup table of a single string for each RPC defined in code. Stripping this out is a micro optimization.

Reply:
To be more clear, I don't see why this shouldn't go into develop as-is. Do you have concerns about the stability?

I think we should just remove the comment to avoid confusion.

Reply:
IDK, I think it's safe to have it all the time and fairly simple/low-cost. I agree with you here.
In terms of general stability, I think it looks good and I couldn't see any issues so far (might do the last check when it's ready to be merged).

Reply:
I'm fine with putting as is. I was under the impression we wanted those only in the case the tools package was there. But if that's not the case, I'll remove those.